Code/Editor/Core/Widgets/PrefabEditVisualModeWidget.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzToolsFramework/Prefab/PrefabFocusNotificationBus.h>
+#include <AzQtComponents/Components/Widgets/ComboBox.h>
+
+#include <QComboBox>
+#include <QLabel>
+#include <QWidget>
+
+// Widget to select the Prefab Edit mode visualization.
+class PrefabEditVisualModeWidget
```

Comment:
Just curious. Why does this need to go in Editor/Core instead of in AzQtComponents?

Reply:
AzQtComponents contains widgets that can be used as a base for UI elements - so they are not tied to specifics (like for example the Prefab system, in this case). This widget is specific to the prefab edit visualization, so it should live in the Editor side of things.